prboom2/src/SDL/i_sound.c

```diff
@@ -227,6 +227,23 @@ static int addsfx(int sfxid, int channel, const unsigned char *data, size_t len)
   return channel;
 }
 
+static int getBufferSize(void)
+{
+  int limit, i;
+
+//if (snd_samplecount) return snd_samplecount;
```

Comment:
I like this solution. Please change the default value for `snd_samplecount` to `0` in `m_misc.c`.